manual/C/ch_Finance-Quote.xml

```diff
@@ -240,6 +251,199 @@ SYNOPSIS
+      </para>
+      
+      <tip>
+        <para>With <userinput>gnc-fq-dump</userinput> you can check symbols faster than from &app;
+          if an error occurs during the retrieval with &app;, because the output of the script
+          is better suited for analysis and not all working symbols are retrieved each time,
+          if you use <guibutton>Get Quotes</guibutton> in the <xref linkend="tool-price" />.
+        </para>
+      </tip>
+
+      <itemizedlist>
+        <listitem>
+          <para>To get verbose information about a security, run a command from the
+            command line of the form:
+            <cmdsynopsis>
+              <command>gnc-fq-dump</command>
+              <arg choice="opt">-v</arg>
+              <arg choice="plain"><replaceable>Source</replaceable></arg>
+              <arg choice="plain" rep="repeat"><replaceable>Symbol</replaceable></arg>
+            </cmdsynopsis>
+            e.g. <command>gnc-fq-dump -v yahoo_json IBM</command>.
+          </para>
+        </listitem>
+        
+        <listitem>
+          <para>To retrieve currency exchange rate run the following command:
+            <command>gnc-fq-dump <optional>-v</optional> currency USD EUR</command>
+            <footnote>
+              <para>Since &app-fq; 1.41 the default source for currencies is
+                hardcoded to be <quote>Alpha Vantage</quote>. Also read the
+                notes on <xref linkend="gnc-tbl-fq-currency-source" />.
+              </para>
+            </footnote>
+            <footnote>
+              <para>The old yahoo currencies are still available as
+                <command>gnc-fq-dump yahoo_json USDEUR=X</command>.
+              </para>
+            </footnote>
+          </para>
+        </listitem>
+      </itemizedlist>
+
+      <para>To test that &app-fq; is working for currencies within &app;,
+        <orderedlist numeration="arabic">
+          <listitem>
+            <para>find a transaction between the desired commodity and the book currency and</para>
+          </listitem>
+          <listitem>
+            <para>right-click on it, then</para>
+          </listitem>
+          <listitem>
+            <para>select &gmi.ac.ed-ex; in the context menu.</para>
+          </listitem>
+        </orderedlist>
+        The <xref linkend="trans-win-enter" /> will appear. Click the 
+        <guilabel>Fetch Rate</guilabel> button and if everything is
+        behaving itself the current rate will fill in the exchange rate box.
+      </para>
+    </sect2>
+
+    <sect2 id="gnc-fq-helper">
+      <title>gnc-fq-helper</title>
+
+      <para>This is a script that is used to retrieve quotes and does not need to be
+        operated by the user. Should the retrieval of a course in &app; result in
+        failures, this script can be used to perform diagnostics and help developers
+        decide if the error is in &app; or &app-fq;.
```

Comment:
s/is used/&app; uses/
s/does not need to be operated by the user./not usually useful for the user to run./
s/course/symbol/